python/delta_sharing/delta_sharing.py

```diff
@@ -150,11 +150,7 @@ def load_as_spark(
     except ImportError:
         raise ImportError("Unable to import pyspark. `load_as_spark` requires PySpark.")
 
-    spark = SparkSession.getActiveSession()
-    assert spark is not None, (
```

Comment:
do we still want to keep the assert?

Reply:
It will not be beneficial to keep it. If any case occurs with issues in spark session fetching then this assert will handle that.

Reply:
I want to hear your thoughts as to what should be the better approach.

Reply:
Sorry for the delay, I'm reading this, it seems it will default to `None`, so I think this assert is still helpful.  https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/SparkSession.html